swagger.yaml

```diff
@@ -611,6 +611,61 @@ paths:
               schema:
                 $ref: "#/components/schemas/Error"
 
+  /v1/appointments/{appointmentId}/{patientId}:
```

Comment:
Change this to /v1/appointments/{appointmentId}/patient

We can fetch the patient ID from keycloak so no need to pass the patientId as a path parameter